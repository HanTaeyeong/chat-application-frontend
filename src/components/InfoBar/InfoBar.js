import React from 'react';

import './InfoBar.scss'
import onlineIcon from '../../icons/onlineIcon.png'
import closeIcon from '../../icons/closeIcon.png'


export default function InfoBar({room,users}) {
    
    return <div className='infobar-outer-container'>
        <div className='infobar-left-inner-container'>
            <img className='infobar-online-icon' src={onlineIcon} alt="online-icon" />
            <h3>{room+" "} </h3>
        </div>
        <h5>{users.length}</h5>
        <div className='infobar-right-inner-container'>
            
            <a href='/'><img src={closeIcon} alt='close-icon'/></a>
        </div>
    </div>;
}
