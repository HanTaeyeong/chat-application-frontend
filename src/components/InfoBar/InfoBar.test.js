import React from 'react'
import InfoBar from './InfoBar'

import {create} from 'react-test-renderer'


describe('InfoBar Component Test', () => {
    test('render test', () => {   
        let app=create(<InfoBar />) 
        expect(app.toJSON()).toMatchSnapshot()
    })
})