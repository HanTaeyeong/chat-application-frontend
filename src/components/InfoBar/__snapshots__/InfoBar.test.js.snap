// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBar Component Test render test 1`] = `
<div
  className="infobar-outer-container"
>
  <div
    className="infobar-left-inner-container"
  >
    <img
      alt="online-icon"
      className="infobar-online-icon"
      src="onlineIcon.png"
    />
    <h3 />
  </div>
  <h5 />
  <div
    className="infobar-right-inner-container"
  >
    <a
      href="/"
    >
      <img
        alt="close-icon"
        src="closeIcon.png"
      />
    </a>
  </div>
</div>
`;
